source/application/transforms/react/index.js

```diff
@@ -13,37 +13,28 @@ export default function createReactCodeFactory(application) {
 			// FIX #13: Merge the environment options with the global options
 			let opts = merge({}, config.opts, configuration.opts);
 
-			let result = convertCode(file, configuration.resolveDependencies !== false, opts);
-			let helpers;
-			let requireBlock;
-			if (configuration.resolveDependencies !== false) {
-				helpers = buildExternalHelpers(undefined, 'var');
-				requireBlock = createRequireBlock(getDependencies(file), true, opts);
-			} else {
-				helpers = '';
-				requireBlock = '';
-			}
+			let result = convertCode(file, configuration.resolveDependencies, opts);
+			const helpers = configuration.resolveDependencies ? buildExternalHelpers(undefined, 'var') : '';
```

Comment:
Are you sure this is the same semantics as before?
The default is configuration.resolveDependencies is not defined (as this doesn't existed before). Now it need to be defined, before if must have been set to false explicitly.